packages/web/src/capture/styles.css

```
@@ -0,0 +1,135 @@
+}
+
+/* dark neutrals — explicit choice */
+:root[data-mode="dark"] {
+  --bg: #201f1d;
+  --surface: #2a2825;
+  --seg: #302d29;
+  --ink: #ece8e1;
+  --ink-soft: #b4afa5;
+  --ink-faint: #8b867c;
+  --line: #37342f;
+  color-scheme: dark;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)